src/components/usa-details/details.stories.js

```diff
@@ -0,0 +1,102 @@
+import "./index";
+import { html, nothing } from "lit";
+
+export default {
+  title: "Components/Details",
+  component: "usa-details",
+  argTypes: {
+    groupName: { name: "Details group name" },
+    bordered: { name: "Add border to panels" },
+    item1Summary: {name: "Item 1 - Summary text"},
+    item1Content: { name: "Item 1 - Panel content"},
+    item1Open: { name: "Item 1 - Open panel on load"},
+    item2Show: { name: "Include item 2 in preview"},
+    item2Summary: {name: "Item 2 - Summary text", if: { arg: 'item2Show' } },
+    item2Content: { name: "Item 2 - Panel content", if: { arg: 'item2Show' } },
+    item2Open: { name: "Item 2 - Open panel on load", if: { arg: 'item2Show' } },
+    item3Show: { name: "Include item 3 in preview"},
+    item3Summary: {name: "Item 3 - Summary text", if: { arg: 'item3Show' } },
+    item3Content: { name: "Item 3 - Panel content", if: { arg: 'item3Show' } },
+    item3Open: { name: "Item 3 - Open panel on load", if: { arg: 'item3Show' } },
```

Comment:
**Thought:** It'd be nice if we could do this in a more intuitive way that wasn't focused on a fixed number of elements.


**Possible control ideas**
1. A button in controls that toggles fields for header/content.
2. An array of objects for header/body content.
3. A textarea to input any number of `<details>`.